src/defs/metrics.d.ts

```diff
@@ -154,6 +154,13 @@ declare module "tsdDef" {
         annotate(key:string, value:string): void;
```

Comment:
This should either be renamed or marked deprecated and the consistently named version added.